{{ cookiecutter.dir_name }}/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
+    }
+}
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
```

Comment:
Can this be evaluated or discovered from the environment? When I tested this, I had 25.0.1 locally, and 25.0.4 was the most current; it wasn't obvious how I could download 25.0.2.